src/subtitle-button/subtitle-button.js

```diff
@@ -50,3 +59,16 @@ Player.translate("disable_closed_captioning",{
 Player.translate("none",{
     en: "None"
 });
+
+Player.translate("subtitle_on_expanded",{
+  en: "Captions are turned on. Expanded subtitle popup button. Use tab to navigate between the options. Press enter key to collapse."
```

Comment:
We should make an effort separately to translate these to our covered languages.

Reply:
should I create a ticket for it?